def foo(a: int):
    return a + 1

foo()     ##ERROR 1 argument expected, 0 provided

foo('a')  ##ERROR first argument should be int
